tools/tsp-client/src/commands.ts

```diff
@@ -311,13 +312,19 @@ export async function generateCommand(argv: any) {
 
 export async function compareCommand(argv: any, args: string[]) {
   let outputDir = argv["output-dir"];
-  const openApiDiffPath = await getPathToDependency("@azure-tools/rest-api-diff");
-  const command = [openApiDiffPath, ...args];
-  try {
-    await nodeCommand(outputDir, command);
-  } catch (err) {
-    Logger.error(`Error occurred while attempting to compare: ${err}`);
-    process.exit(1);
+  if (argv["update"]) {
+    execSync(`npm install @azure-tools/rest-api-diff@latest`, { stdio: "inherit" });
```

Comment:
We were just discussing this with @mikeharder in this other PR: #9602. I dont think we need to have this functionality in tsp-client. It seems if a user is at the point of knowing that they need to update the package version then they should simply run the npm update command directly.

Reply:
Can this be as simple as updating your output message to be the following:

_This tool is under active development. If you experience issues or have questions, please contact Travis Prescott direct
ly ([trpresco@microsoft.com](mailto:trpresco@microsoft.com)). [Tool version: 0.1.8]
**Run `npm update @azure-tools/rest-api-diff` to update @azure-tools/rest-api-diff to the latest version**_

Reply:
I believe `npm update @azure-tools/rest-api-diff@latest` should work as well to be very explicit but if it doesnt then npm install should be fine.

Reply:
Well, the problem is from what folks have told me, running npm update doesn't actually work. It still uses the original internal version that tsp-client originally pulled down.

Reply:
I have had issues with npm update myself...maybe just recommending npm install, the same as the command you're wrapping under this command would be better?